environment.yml

```diff
@@ -8,11 +8,11 @@ dependencies:
   - litex-hub::icestorm=0.0_0719_g792cef0=20201120_145821
   - litex-hub::iverilog=s20150603_0957_gad862020=20201120_145821
   - litex-hub::openocd=0.10.0_1514_ga8edbd020=20201119_154304
-  - litex-hub::symbiflow-yosys-plugins=1.0.0_7_284_gb6c5f5d=20210318_102115
+  - litex-hub::symbiflow-yosys-plugins=1.0.0_7_305_g9e969e2=20210415_092356
   - litex-hub::prjxray-tools=0.1_2842_g6867429c=20210301_104249
   - litex-hub::prjxray-db=0.0_248_g2e51ad3=20210312_125539
   - litex-hub::vtr-optimized=8.0.0_3452_ge7d45e013=20210318_102115
-  - litex-hub::yosys=0.9_5266_g0fb4224e=20210301_104249_py37
+  - litex-hub::symbiflow-yosys=0.9_0_g40d9e120=20210413_171008_libffi33  
```

Comment:
This should not be `symbiflow-yosys`, but rather `yosys` only.

For reference, the [`symbiflow-yosys`](https://github.com/hdl/conda-eda/blob/master/syn/symbiflow-yosys/meta.yaml) package builds the `master+wip` branch symbiflow fork of yosys, which is outdated and not in line with the current yosys main branch, as we moved to upstream yosys.